controllers/repo_manager/controller.go

```diff
@@ -294,7 +294,8 @@ func (r *RepoManagerReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			} else if pulpController.Requeue {
 				return pulpController, nil
 			}
-		} else {
+		}
+		if needsPulpWeb {
```

Comment:
@git-hyagi from what I tested this is the only change required to make the ingress run with pulp web (when ingress isn't nginx).

About the certificate, I notice even when nginx ingress, we aren't supporting TLS